installer.sh

```diff
@@ -493,10 +524,10 @@ case ${ANSWER:0:1} in
         y|Y )
           sudo cve_searchsploit -u
```

Comment:
do we really need this sudo here?
Should we run such an update also outside of the docker environment. Otherwise the first run will do it and it will take a while during the first emba run.